components/ratgdo/sensor/__init__.py

```diff
@@ -38,3 +40,15 @@ async def to_code(config):
     await cg.register_component(var, config)
     cg.add(var.set_ratgdo_sensor_type(config[CONF_TYPE]))
     await register_ratgdo_child(var, config)
+
+    if config['type'] == 'distance':
```

Comment:
Probably needs `black` run on all the `python` code as I'm not sure `pre-commit` does it automatically

Reply:
I took care of adding it to pre-commit so the CI will take care of it automatically now once this PR is up to date with `main`